README.md

```diff
@@ -1,3 +1,41 @@
+# A derived version of node-red-contrib-blockly
+
+This is a derived version of node-red-contrib-blockly release-1.1.0 @ https://github.com/bartbutenaers/node-red-contrib-blockly
+This version is a progressing experiment with upgraded Blockly, expanded Blockly editor, import/export XML library and etc.
+
```

Comment:
Can you please remove this section on the readme, since it is related to your fork.

Reply:
Done. :)